src/lib/mod.rs

```diff
@@ -206,13 +206,15 @@ pub fn get_window_at_point(
     win: xcb::Window,
     pt: xcb::Point,
     remove_decorations: u32,
+    ignore_window: Option<xcb::Window>,
 ) -> Option<HacksawResult> {
     let tree = xcb::query_tree(conn, win).get_reply().unwrap();
     let children = tree
         .children()
         .iter()
         .filter(|&child| viewable(conn, *child))
         .filter(|&child| input_output(conn, *child))
+        .filter(|&child| ignore_window.map_or(true, |window| *child != window))
```

Comment:
This line is a little hard to follow - could we make it more clear?

Reply:
Do you have any idea how to make it clearer?

Reply:
Perhaps explicitly `if let Some(w) = ignore_window { *child != window } else { true }`. But `map_or` is kinda nicer if you know what it does.

Reply:
That's exactly the same as `map_or`.

Reply:
Yes, I know, but I put it because @expectocode wanted the same behaviour in a "less confusing way". We should wait on their thoughts.

Reply:
We could do `filter(|&child| Some(*child) != ignore_window)`

Reply:
to remove the weird `*child`, we could take `ignore_window: Option<&xcb::Window>`

Reply:
"Keep unless `ignore_window` is `Some(*child)`", so if it's `None` it's kept as well. Eh, takes a bit of thought still but I do find the `map_or(true, ...)` a bit weird.

Reply:
I think it's likely that we'll change the type of ignore_window Option<Window> to just Window in the near future tbh, so this will all be moot

Reply:
Ah, I didn't even thought about that, yeah `Some(*child) != ignore_window` is definitely better.